storagenode/storagenodedb/storagenodedbtest/run_test.go

```diff
@@ -30,6 +31,83 @@ func TestDatabase(t *testing.T) {
 	})
 }
 
+func TestBandwidthRollup(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	log := zaptest.NewLogger(t)
+
+	db, err := storagenodedb.NewInMemory(log, ctx.Dir("storage"))
```

Comment:
I wonder if the locking issues we had were because we always test the inmemory version vs the disk